content/workshops/rhel_8/exercise1.2.adoc

```diff
@@ -0,0 +1,158 @@
+---
+title: "Exercise 1.2 - Managing Software in an Application Stream"
+workshops: rhel_8
+workshop_weight: 12
+layout: lab
+---
+
+:icons: font
+:imagesdir: /workshops/rhel_8/images
+:package_url: http://docs.ansible.com/ansible/package_module.html
+:service_url: http://docs.ansible.com/ansible/service_module.html
+:gather_facts_url: http://docs.ansible.com/ansible/latest/playbooks_variables.html#turning-off-facts
+
+
+= Exercise 1.2 - Managing Software in an Application Stream
+
+Application streams are a new capability, in RHEL 8, providing access to the latest tools, languages and libraries, without affecting core system components. This is done by including all of those `streams` in containers. The containers are automatically instantiated, when needed, and go away, when they are not in use.
```

Comment:
Line 17 Application streams are a new RHEL 8 capability that provides access to the latest tools, languages and libraries, without affecting core system components. ……

Reply:
Fixed